app/src/sharedTest/java/org/oppia/app/options/OptionsFragmentTest.kt

```diff
@@ -194,6 +196,48 @@ class OptionsFragmentTest {
     }
   }
 
+  @Test
+  fun testOptionFragment_clickDefaultAudioLanguage_configurationChange_changeDefaultAudioLanguageToEnglishSuccessfully() {
+    launch<OptionsActivity>(createOptionActivityIntent(0)).use {
+      onView(isRoot()).perform(orientationLandscape())
+      onView(atPositionOnView(R.id.options_recyclerview, 2, R.id.audio_laguage_item_layout)).perform(
+        click()
+      )
+
+      onView(withId(R.id.audio_language_recycler_view))
+        .perform(
+          RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(
+            1,
+            click()
+          )
+        )
+      onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
+      onView(atPositionOnView(R.id.options_recyclerview, 2, R.id.audio_language_text_view)).check(
+        matches(withText("English"))
+      )
+    }
+  }
+
+    @Test
+  fun testOptionFragment_clickDefaultAudioLanguage_configurationChange_changeDefaultAudioLanguageToChineseSuccessfully() {
```

Comment:
These two test cases are actually not helpful as you are not making any orientation change in between any clicks/selections from users.

You should write something like this:
`testOptionFragment_checkDefaultAudioLanguage_changeLanguageToHindi_changeConfiguration_checkHindiLanguageIsSelected`

Reply:
> @chrk2205 Please keep track of assignees section by following these guidelines.
> 
> https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section

@rt4914  sorry it was by miss it wont happen again!!

Reply:
> These two test cases are actually not helpful as you are not making any orientation change in between any clicks/selections from users.
> 
> You should write something like this:
> `testOptionFragment_checkDefaultAudioLanguage_changeLanguageToHindi_changeConfiguration_checkHindiLanguageIsSelected`

  @Test
  fun testOptionFragment_checkDefaultAudioLanguage_changeLanguageToEnglish_changeConfiguration_checkEnglishLanguageIsSelected{
    launch<OptionsActivity>(createOptionActivityIntent(0)).use {
      onView(atPositionOnView(R.id.options_recyclerview, 2, R.id.audio_laguage_item_layout)).perform(
        click()
      )

      onView(withId(R.id.audio_language_recycler_view))
        .perform(
          RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(
            1,
            click()
          )
        )
      onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
      onView(isRoot()).perform(orientationLandscape())
      onView(atPositionOnView(R.id.options_recyclerview, 2, R.id.audio_language_text_view)).check(
        matches(withText("English"))
      )
    }
  }

does this look good for that functtion @rt4914